include/ipmitool/ipmi.h

```diff
@@ -250,6 +250,7 @@ struct ipmi_rs {
 #define IPMI_BMC_SLAVE_ADDR		0x20
 #define IPMI_REMOTE_SWID		0x81
 
+#define IPMI_NETFN_QCTOEM		0x36
```

Comment:
This doesn't belong here and must go to `include/ipmitool/ipmi_quantaoem.h` just like it is done for sunoem, et al.